tools/universe/package_builder.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
-_jre_url = "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz"
+_jre_url = "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jdk_x64_linux_hotspot_8u212b03.tar.gz"
```

Comment:
Hi dear @kaiwalyajoshi, @samvantran,
I just noticed that previously **JRE** was used here and in other places, but now it's pointing to **JDK**, although I guess make sense switch it to **JRE**.  You can find it here https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u212-b03/OpenJDK8U-jre_x64_linux_hotspot_8u212b03.tar.gz

Reply:
We're using `https://downloads.mesosphere.com/java/openjdk-8u212b03-hotspot-linux-x64.tar.gz` to maintain consistency with the other named packages.

The two images are exactly the same, just named differently.

Reply:
After editing your the message, LGTM:-)